duet/explores/desktop_activation.explore.lkml

```diff
@@ -0,0 +1,74 @@
+include: "../views/*.view.lkml"
+
+explore: desktop_activation {
+  sql_always_where: ${submission_timestamp_date} > date(2020, 7 ,1) AND
+    ${channel} = "release" AND
+    DATE_DIFF(  -- Only use builds from the last month
+      ${submission_timestamp_date},
+      SAFE.PARSE_DATE('%Y%m%d', SUBSTR(${build_id}, 0, 8)),
+      MONTH
+    ) <= 1 AND
+    ${os} = "Windows" AND
+    ${attribution_source} IS NOT NULL AND
+    ${distribution_id} IS NULL AND
+    ${attribution_ua} != "firefox" AND
+    ${startup_profile_selection_reason} = "firstrun-created-default" AND
+    DATE(${submission_timestamp_date}) <= DATE_SUB(
+      IF({% parameter desktop_activation.previous_time_period %},
+        -- if the data for the previous time period is requested,
+        -- shift dates by the date range provided via the 'date' filter
+        DATE(DATE_ADD(
+          DATE({% date_end desktop_activation.date %}), INTERVAL DATE_DIFF(DATE({% date_start desktop_activation.date %}), DATE({% date_end desktop_activation.date %}), DAY) DAY)),
+          DATE({% date_end desktop_activation.date %})),
+      -- if the most recent week is to be ignored, shift date range by 8 days
+      INTERVAL IF({% parameter desktop_activation.ignore_most_recent_week %}, 8, 0) DAY)
+    AND
+    DATE(${submission_timestamp_date}) > DATE_SUB(
+      IF({% parameter desktop_activation.previous_time_period %},
+      -- if the data for the previous time period is requested,
+      -- shift dates by the date range provided via the 'date' filter
+      DATE(DATE_ADD(
+        DATE({% date_start desktop_activation.date %}), INTERVAL DATE_DIFF(DATE({% date_start desktop_activation.date %}), DATE({% date_end desktop_activation.date %}), DAY) DAY)),
+        DATE({% date_start desktop_activation.date %})),
+      -- if the most recent week is to be ignored, shift date range by 8 days
+      INTERVAL IF({% parameter desktop_activation.ignore_most_recent_week %}, 8, 0) DAY);;
+  join: country_buckets {
+    type: cross
+    relationship: many_to_one
+    sql_where: ${country_buckets.code} = ${desktop_activation.normalized_country_code} ;;
+  }
+  always_filter: {
+    filters: [
+      desktop_activation.date: "28 day",
+      desktop_activation.ignore_most_recent_week: "yes"
+    ]
+  }
+
+  aggregate_table: rollup__country_buckets_bucket__submission_date {
+    query: {
+      dimensions: [country_buckets.bucket, desktop_activation.submission_timestamp_date]
+      measures: [desktop_activation.activations]
+      filters: [desktop_activation.date: "28 days", desktop_activation.ignore_most_recent_week: "Yes"]
```

Comment:
It doesn't look like Looker is using these because the dashboard filters on `"28 day"`, while this has `"28 days"`. Did you use the "generated aggregate table LookML" feature? If you did, there must be a bug in the implementation.

Try switching to `"28 day"` and we can check to see if the aggregate tables are used.

FYI, I checked by looking at the running queries in Looker. I saw this in the issued query:
```
-- Did not use desktop_activation::rollup__country_buckets_bucket__submission_date; filter field desktop_activation.date did not have the same filter value in the query ('28 day') as in the aggregate_table ('28 days')
-- Did not use desktop_activation::rollup__country_buckets_bucket__submission_date_prev; filter field desktop_activation.date did not have the same filter value in the query ('28 day') as in the aggregate_table ('28 days')
```

Now, you can do the same by inspecting queries from the BQ console. These queries all run in the mozdata project, so you can look through and find them to see if Looker is using the aggregate tables or the cache.

Reply:
I fixed the filter and looked into the BigQuery logs. It's now using the cached tables